bitnami/schema-registry/templates/statefulset.yaml

```diff
@@ -271,11 +271,19 @@ spec:
           configMap:
             name: {{ include "schema-registry.log4j.configmapName" . }}
         {{ end }}
-        {{- if contains "SSL" $kafkaProtocol }}
+        {{- if or (contains "SSL" $kafkaProtocol) .Values.auth.tls.enabled }}
         - name: certificates
-          secret:
-            secretName: {{ printf "%s" (tpl .Values.auth.kafka.jksSecret $) }}
-            defaultMode: 256
+          projected:
+            defaultMode: 0400
+            sources:
+            {{- if contains "SSL" $kafkaProtocol }}
+            - secret:
+                name: {{ printf "%s" (tpl .Values.auth.kafka.jksSecret $) }}
+            {{ end }}
```

Comment:
Looks good to me, but picky comment, could you please use `{{- end }}` instead of  `{{ end }}`?
The `{{end}}` will render an extra empty line

Reply:
Thanks! PR is now updated